about.html

```diff
@@ -0,0 +1,181 @@
+
+            <div class="main-nav">
+                <div class="subNav d-flex">
+                    <div class="subLeft">
+                        <img src="./Images/logo.jpg" alt="">
+                    </div>
+                    <div class="subMiddle d-flex">
+                        <a href="index.html" class="active">Home</a>
+                        <a href="">Program</a>
+                        <a href="">Join</a>
+                        <a href="">Sponsor</a>
+                        <a href="">News</a>
+                    </div>
+                    <div class="subRight">
+                        <a href="#">
+                            CC campaign
+                        </a>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div class="nav-menu">
+            <img src="images/hamburger.svg" class="hamburger" id="menu"
+                alt=" The menu bar for easy access to Portfolio's content">
+            <ul class="menuItems" id="item">
+                <!-- <img class="closemenu" id="close" src="./Images/close.svg" alt=""> -->
+
+                <div class="listCan" id="list">
+                    <li><a href="index.html" class="item" id="first">Home</a></li>
+                    <li><a href="" class="item">Program</a></li>
+                    <li><a href="" class="item">Join</a></li>
+                    <li><a href="" class="item">Sponsor</a></li>
+                    <li><a href="" class="item">News</a></li>
+                    <li>
+                        <div class="subRight">
+                            <a href="#">
+                                CC campaign
+                            </a>
+                        </div>
+                    </li>
+                </div>
+            </ul>
+        </div>
+    </nav>
+
+    <header>
+        <div class="headerTextContent">
+            <h1>
+                This is talking about<br> CELEBRITY FEUD 2023.
+            </h1>
+            <div class="header_p">
+                <p>
+                    Throughout this session,
+                    we'll get to know some famous celebrities,
+                    how they got to where they are,<br>
+                    and what motivation their experience might give us.
+                    Getting to know the big guns in town is very important.
+                    Throughout this session,
+                    we'll get to know some famous celebrities,
+                    how they got to where they are,<br>
+                    and what motivation their experience might give us.
+                    Getting to know the big guns in town is very important.
+                    Throughout this session,
+                    we'll get to know some famous celebrities,
+                    how they got to where they are,<br>
+                    and what motivation their experience might give us.
+                    Getting to know the big guns in town is very important.
```

Comment:
I recommend addressing the following issue with this section as they can make your project more attractive;

- The font size of the paragraph within the box, is quite smaller than that of the design
- The colour for the call to action text with `@` is blue which is not the same as the template 
- The paddings to the left and right are a bit too much, consider reducing

Here is a screenshot with the areas of concern highlighted in blue

<img width="238" alt="space" src="https://github.com/Wils-FOURR-speed/Capstone-1/assets/84930355/e1e53f33-ab44-4f07-b950-61a60383fe09">